libs/components/packages/src/schematics/migrations/update-10/ag-grid/ag-grid.schematic.spec.ts

```diff
@@ -1,7 +1,13 @@
 import { Tree } from '@angular-devkit/schematics';
 import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
 
-const UPDATE_TO_VERSION = '30.0.0';
+import fs from 'fs-extra';
+import { joinPathFragments } from 'nx/src/utils/path';
+import { workspaceRoot } from 'nx/src/utils/workspace-root';
+
+const UPDATE_TO_VERSION = fs.readJsonSync(
+  joinPathFragments(workspaceRoot, 'package.json'),
+).dependencies['ag-grid-community'];
```

Comment:
Reducing the places where the version is hard-coded.

Reply:
Same feedback as above.

Reply:
Same. I switched to a single test that verifies the version tested is the same as the dependency.